docker/docker-compose.extend.yml

```diff
@@ -11,7 +11,7 @@ services:
             - /workspace/__pycache__
             - /workspace/.vite
             - /workspace/.yarn/unplugged
-            - app-assets:/assets
+            - /Users/lars/Documents/IETF/assets:/assets
```

Comment:
Well, no. But I understand the desire to have some sort of local persistent store to use rather than suffering the resync every time.

Reply:
Sorry, that was an accidental inclusion.